### PR TITLE
Changes in documentation

### PR DESCRIPTION
**Pull request by kkujawa-sumo**
###### Description

- Corrected formatting
- Added useful information to vagrant readme

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: vagrant/README.md [hunk 1]
```diff
@@ -30,20 +30,22 @@ After successfull installation you can ssh to the virtual machine with:
...
+
 ## Build
 
-To perform docker image build and to run tests please use `build` target:
+On the virtual machine, to perform docker image build and to run tests please use `build` target:
 
 ```bash
-/sumologic/vagrant/Makefile build
+sumo-make build
 ```
```
Comment (pmalek-sumo) [inline vagrant/README.md:41]: keep the old one as alternative as well  `... or ...` (2 snippets)

File: vagrant/README.md [hunk 2]
```diff
@@ -30,20 +30,22 @@ After successfull installation you can ssh to the virtual machine with:
...
 
 ## Collector
 
 To install or upgrade collector please type:
 
 ```bash
-/sumologic/vagrant/Makefile upgrade
+sumo-make upgrade
 ```
```
Comment (pmalek-sumo) [inline vagrant/README.md:49]: ditto

File: vagrant/README.md [hunk 3]
```diff
@@ -52,11 +54,25 @@ and after that it will install/upgrade collector in the vagrant environment.
 To remove collector please use:
 
 ```bash
-/sumologic/vagrant/Makefile clean
+sumo-make clean
 ```
```
Comment (pmalek-sumo) [inline vagrant/README.md:58]: ditto

File: vagrant/README.md [hunk 4]
```diff
@@ -52,11 +54,25 @@ and after that it will install/upgrade collector in the vagrant environment.
...
-List of other useful commands in the Makefile:
+List of other useful options for `sumo-make`:
 
 - `expose-prometheus` - exposes prometheus on port 9090 of virtual machine
 - `expose-grafana` - exposes grafana on port 8080 of virtual machine
 - `apply-avalanche` - run one pod deployment of avalanche (metrics generator)
+
+## Test
+
+To quickly test whether sumo-kubernetes-collection works receiver-mock can be used.
```
Comment (pmalek-sumo) [inline vagrant/README.md:68]:
```suggestion
In order to quickly test whether sumo-kubernetes-collection works, one can use `receiver-mock` for that purpose.
```

🤷🏻

File: vagrant/README.md [hunk 5]
```diff
@@ -52,11 +54,25 @@ and after that it will install/upgrade collector in the vagrant environment.
...
+
+To check receiver-mock logs please use:
+```
+kubectl logs $(kubectl get pod -l app=receiver-mock -o jsonpath="{.items[0].metadata.name}"  -n receiver-mock) -n receiver-mock
+```
+To check metrics exposed by receiver-mock please use:
+```
+kubectl exec $(kubectl get pod -l app=receiver-mock -o jsonpath="{.items[0].metadata.name}"  -n receiver-mock) -it -n receiver-mock -- curl http://localhost:3000/metrics
+```
```
Comment (pmalek-sumo) [inline vagrant/README.md:77]: Empty lines around snippet backticks

File: vagrant/README.md [hunk 6]
```diff
@@ -51,12 +65,35 @@ and after that it will install/upgrade collector in the vagrant environment.
 
 To remove collector please use:
 
+```bash
+sumo-make clean
+
```
Comment (pmalek-sumo) [inline vagrant/README.md:70]:
```suggestion
sumo-make clean
```

Comment (pmalek-sumo) [reply vagrant/README.md:70]: redundant empty line

File: vagrant/README.md [hunk 7]
```diff
@@ -30,18 +30,32 @@ After successfull installation you can ssh to the virtual machine with:
 vagrant ssh
 ```
 
+NOTICE: The directory with sumo-kubernetes-collection repository is synced with the virtual machine.
```
Comment (pmalek-sumo) [inline vagrant/README.md:33]:
```suggestion
NOTICE: The directory with sumo-kubernetes-collection repository on the host is synced with `/sumologic/` directory on the virtual machine.
```

**Comment by kkujawa-sumo** [comment]
@pmalek-sumo please take another look

File: vagrant/README.md (hunk 7 above)
Comment (pmalek-sumo) [reply vagrant/README.md:33]: ping @kkujawa-sumo

File: vagrant/README.md [hunk 8]
```diff
@@ -51,12 +65,34 @@ and after that it will install/upgrade collector in the vagrant environment.
...
+
+```
+kubectl logs $(kubectl get pod -l app=receiver-mock -o jsonpath="{.items[0].metadata.name}"  -n receiver-mock) -n receiver-mock
+```
+
+To check metrics exposed by receiver-mock please use:
+
+```
+kubectl exec $(kubectl get pod -l app=receiver-mock -o jsonpath="{.items[0].metadata.name}"  -n receiver-mock) -it -n receiver-mock -- curl http://localhost:3000/metrics
+```
```
Comment (sumo-drosiek) [inline vagrant/README.md:98]: We could add those commands as new targets to the `Makefile`. Not necessary in this PR :)

Comment (kkujawa-sumo) [reply vagrant/README.md:98]: > We could add those commands as new targets to the `Makefile`. Not necessary in this PR :)

I'll add this in a new PR

**Review by pmalek-sumo — APPROVED** [review]

**Review by sumo-drosiek — APPROVED** [review]

**Pull request merged**